### DISCUSSION TITLE
v0.18.3

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#1313] [Releases]
## What's New
* [Feature] Added packet loss to form view and table by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1300

## What's Changed
* Laravel 10.48.2 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1308
* Release v0.18.3 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/1312

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.18.2...v0.18.3

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.18.3'>v0.18.3</a>.</em>

**Comment by seh2000** [comment]
No problem in upgrading to 0.18.3, the two servers that give me the best DL/UL results don't return the packet loss metric.
Tried a couple not too good servers, they did return packet loss, so new release works.

![Screenshot 2024-03-15 163500](https://github.com/alexjustesen/speedtest-tracker/assets/74658646/befeaa93-2686-4bb1-92d5-643f119b7530)

For export result, Packet Loss not an option to select.

![Screenshot 2024-03-15 163615](https://github.com/alexjustesen/speedtest-tracker/assets/74658646/9d0c50c3-1f88-4137-8a6f-6d97abf3b216)
